README.md

```diff
@@ -7,6 +7,7 @@ Here you can find the most **delicious** recipes to cook delicious meals using o
 ## 🌽 Table of contents
 
 - [Amazon Web Services (AWS) Elastic Beanstalk](/aws-eb)
```

Comment:
Can you update the .github.com/dependabot.yml file? Thanks

Reply:
Thanks, I just updated the .github.com/dependabot.yml file.